src/content-helper/dashboard-widget/top-posts/component-list-item.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TopPostData } from './model';
+import { impreciseNumber } from '../../../blocks/shared/functions';
+import PublishedLinkIcon from '../../../blocks/content-helper/icons/published-link-icon';
+import { getSmartShortDate } from '../../../blocks/shared/utils/date';
+
+interface TopPostListItemProps {
+	post: TopPostData;
+}
+
+/**
+ * Returns a single list item depicting a post.
+ *
+ * @param {TopPostData} post The Post to be shown.
+ */
+function TopPostListItem( { post }: TopPostListItemProps ): JSX.Element {
+	return (
+		<li className="parsely-top-post">
+			<div className="parsely-top-post-wrapper">
```

Comment:
Using `wrapper` makes more sense if we uses this `div` to wrap around `li` .... I think `parsely-top-post-content` is a better name.